tests/test_cli.py

```diff
@@ -39,20 +45,20 @@ def test_init(tmp_path: pathlib.Path):
     assert os.path.exists(path_script_predict)
 
 
-def test_train_model_create(tmp_path: pathlib.Path):
+@pytest.mark.usefixtures('project_dir')
+def test_train_model_create():
     """Verify that 'fnet train' creates default jsons."""
-    _init_test(tmp_path)
-    path_create = os.path.join('test_model', 'train_options.json')
+    path_create = os.path.join('created', 'train_options.json')
```

Comment:
This can be done in `pathlib.Path` if you wanted to unify your path library.

`path_create = Path("created", "train_options.json")`